scripts/observers/start.py

```diff
@@ -1,41 +1,41 @@
-import os
-import sys
-from pathlib import Path
 from dotenv import load_dotenv
+from utils import *
 
-load_dotenv()
 
-working_dir = str(Path.home()) + str(os.getenv('WORKING_DIRECTORY'))
-observer_dir = str(os.getenv('OBSERVER_DIR'))
-
-
-def start_observer(shard_id):
-    current_observer = observer_dir + str(shard_id)
-    working_dir_observer = working_dir + current_observer
+def start_observer(shard_id, working_dir):
+    current_observer = str(os.getenv('OBSERVER_DIR_PREFIX')) + str(shard_id)
+    working_dir_observer = Path(working_dir / current_observer)
 
     current_directory = os.getcwd()
     # start observer
-    os.chdir(working_dir_observer + "/node")
+    os.chdir(Path(working_dir_observer / "node"))
```

Comment:
The cast to `Path` is not required (here and below).

Reply:
removed